src/mochiweb_http.erl

```diff
@@ -65,9 +67,11 @@ stop(Name) ->
 %%      The proplist is as follows: [{name, Name}, {port, Port}, {active_sockets, ActiveSockets}, {timing, Timing}].
 %% @end
 start(Options) ->
+    {ok, _Pid} = mochiweb_clock:start(),
```

Comment:
I have a feeling that tying a globally clock caching service directly to non-global web server instance may cause problems for people. It might be better to just have some sort of "ensure started" check in start/start_link that will start (not start_link) the clock and leave it orphaned.